File: csharp/ex018.cs
// Faça um programa que leia o ano de nascimento de uma pessoa, calcule a idade
// dela e depois mostre se ela pode ou não votar.


class Ex018
{
  public int yearBirth;

  public void CouldIVote()
  {
    Console.WriteLine("Digite em qual ano você nasceu: ");
    yearBirth = Convert.ToInt32(Console.ReadLine());

    DateTime actualYear = new DateTime();
    int agePerson = actualYear.Year - yearBirth;


    Console.Clear();
    if (agePerson >= 16)
    {
      Console.WriteLine($"Você tem {agePerson} anos de idade, já pode votar");
    }
    else
    {
      Console.WriteLine($"Você tem {agePerson} anos de idade, ainda não pode votar");
    }
    
  }
}

File: python/ex018.py
# Faça um programa que leia o ano de nascimento de uma pessoa, calcule a idade
# dela e depois mostre se ela pode ou não votar.

import datetime

yearBirth = int(input("Digite em qual ano você nasceu: "))

actualYear = datetime.datetime.now()
agePerson = actualYear.year - yearBirth

if (agePerson >= 16):
    print(f"Você tem {agePerson} anos de idade, já pode votar")
else:
    print(f"Você tem {agePerson} anos de idade, ainda não pode votar")
